web/src/main/java/org/cbioportal/web/MutationController.java

```diff
@@ -78,7 +78,9 @@
         @ApiParam("Name of the property that the result list is sorted by")
         @RequestParam(required = false) MutationSortBy sortBy,
         @ApiParam("Direction of the sort")
-        @RequestParam(defaultValue = "ASC") Direction direction) throws MolecularProfileNotFoundException {
+        @RequestParam(defaultValue = "ASC") Direction direction,
+        @ApiParam("Annotation")
+        @RequestParam(required = false) String annotation) throws MolecularProfileNotFoundException {
```

Comment:
We can now remove the annotation in this endpoint since it was a testing project.